src/access/universal.test.ts

```diff
@@ -1084,3 +1086,96 @@ describe("setPublicAccess", () => {
     expect(access).toBeNull();
   });
 });
+
+describe("getAccessFor", () => {
+  it("calls to getAgentAccess with the appropriate parameters", async () => {
+    const universalModule = jest.requireActual("./universal") as {
+      getAgentAccess: () => Promise<Access | null>;
+    };
+    universalModule.getAgentAccess = jest.fn();
+    const options = {
+      fetch: jest.fn().mockResolvedValue({
+        ok: false,
+      } as never) as typeof fetch,
+    };
+    await getAccessFor(
+      "https://some.resource",
+      "agent",
+      "https://some.pod/profile#webid",
+      options
+    );
+    expect(universalModule.getAgentAccess).toHaveBeenCalledWith(
+      "https://some.resource",
+      "https://some.pod/profile#webid",
+      options
+    );
+  });
+
+  it("throws if the agent has been ommited", async () => {
```

Comment:
```suggestion
  it("throws if the agent has been omitted", async () => {
```

Reply:
Dangit, another false friend ! To this day, I can't write professional in English properly on the first try because it has a double n in French.
6a18b8484ceb95bf1a0c72ad9e8f455c48a422be